src/libs/runtime-utils.js

```diff
@@ -222,33 +223,33 @@ export const machineCost = machineType => {
   return _.find(knownMachineType => knownMachineType.name === machineType, machineTypes).price
 }
 
-export const getCurrentApp = _.flow(trimAppsOldestFirst, _.last)
+export const getCurrentAppForType = appType => _.flow(trimAppsOldestFirst, _.filter(_ => _.appType === appType), _.last)
```

Comment:
In Scala, we typically use `_` when we don't care/use an input variable. In Javascript (at least in this code base) we usually use `_` for Lodash library. Therefore, it may be confusing to use `_` like above.

```suggestion
export const getCurrentAppForType = appType => _.flow(trimAppsOldestFirst, _.filter(app => app.appType === appType), _.last)
```
Similarly for other places in this file and `notebook-utils`

Reply:
Sure, fixed these (and did a little simplification at the same time).